docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,9 @@
+= Server-Sent Events
+
+Server-sent events (SSE) is a protocol that you can use to push unidirectional events form an event source to a web-based or browser-based client over HTTP.
+It is more light-weight compared to REST or WebSockets, and it does not require the target client to be aware of the protocol being used.
+As opposed to WebsSockets, it does not handle bidirectional communication beetween an event source 
```

Comment:
As opposed to WebSockets, SSE does...

Reply:
Not sure if you want to do anything with it or not, but the protocol is called WebSocket, not WebSockets. (See also RFC 6455.)